start-database.sh

```diff
@@ -8,7 +8,11 @@ echo "Adding data into MySQL"
 file -bi /var/mysql/database.sql
 /usr/sbin/mysqld &
 sleep 5
-curl $url | mysql --default-character-set=utf8
+curl $url -o import.sql
+sed -ri.bak 's/-- Database: (.*?)/CREATE DATABASE \1/g' import.sql
+if grep -q "CREATE DATABASE" file.sql; then :; else sed -ri.bak 's/-- Table structure/CREATE DATABASE `database_1`\n-- Table Structure/g' import.sql; fi
```

Comment:
I do not understand the necessity of this line here. Why you are insert before every table definition a new CREATE DATABASE statement? There are a plenty of CREATE DATABASE statements in import.sql.

Why yo are grepping 'file.sql'. Where does this file come from? Do you mean 'import.sql'?